changes/zh-cn/2.0.0.md

```diff
@@ -66,6 +66,7 @@ Seata 是一款开源的分布式事务解决方案，提供高性能和简单
 - [[#5748](https://github.com/seata/seata/pull/5748)] 修复在某些情况下，业务sql中主键字段名大小写与表元数据中的不一致，导致回滚失败
 - [[#5745](https://github.com/seata/seata/pull/5745)] 修复不满足 sofa-rpc 中 setAttachment 方法的参数前缀要求问题
 - [[#5772](https://github.com/seata/seata/pull/5762)] 修复TableMetaCache的一些字段类型，避免溢出
+- [[#5794](https://github.com/seata/seata/pull/5794)] 解决redis作为注册中心时cluster无法自定义的BUG
```

Comment:
PR No. is 5787
![image](https://github.com/seata/seata/assets/8758457/7969619f-dfb3-4bb9-a20b-f58c1288a4e3)